css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,18 @@
+input {
+  color: black;
+}
+
+#button:hover {
+  background-color: black;
+  color: white;
+}
+
+.header {
+  background-color: black;
+  color: white;
+  border: 0.5px solid white;
+}
+
+.gray {
+  background-color: lightgray;
+}
```

Comment:
Please accomplish this with a pseudo-class rather than adding classes to individual elements yourself.